src/components/Gallery/utils/getGalleryItemCopyLinkAction.tsx

```diff
@@ -0,0 +1,32 @@
+import {Link} from '@gravity-ui/icons';
+import {ActionTooltip, Button, CopyToClipboard, Icon} from '@gravity-ui/uikit';
+
+import {GalleryItemAction} from '../GalleryItem';
+import {i18n} from '../i18n';
+
+export type GetGalleryItemCopyLinkActionArgs = {
+    copyUrl: string;
+    onCopied?: () => void;
```

Comment:
Commonly past tense is not used for callback names.

```suggestion
    onCopy?: () => void;
```